tests/core/pyspec/eth2spec/test/helpers/merge/fork.py

```diff
@@ -0,0 +1,45 @@
+MERGE_FORK_TEST_META_TAGS = {
+    'fork': 'merge',
+}
+
+
+def run_fork_test(post_spec, pre_state):
```

Comment:
Might be worth abstracting this into a standard `run_fork_test(post_spec, pre_state, stable_fields, modified fields, fork_specific_checks_fn=None)` and have `run_merge_fork_test` call it. But... doesn't matter too much for this PR